pkg/proc/proc_test.go

```diff
@@ -3595,7 +3595,7 @@ func TestIssue1145(t *testing.T) {
 
 func TestDisassembleGlobalVars(t *testing.T) {
 	if runtime.GOARCH == "arm64" {
-		t.Skip("test is not valid on ARM64")
+		t.Skip("there is always no GlobalVars in disass on ARM64")
```

Comment:
say: "On ARM64 symLookup can't look up variables due to how they are loaded, see issue #1778"

Reply:
Fixed.